datasets.json

```diff
@@ -179,5 +179,66 @@
       "release_date": "05-26-2021",
       "license": "CC BY 4.0"
     }
+  },
+  {
+    "name": "Extreme heat",
```

Comment:
It looks like this list ~almost chronologically sorted manually.

Maybe worth adding these new entries to the top of the list (OffsetsDB as first element, then extreme heat)?